app/audit/DeleteEmploymentAudit.scala

```diff
@@ -44,5 +45,38 @@ case class DeleteEmploymentAudit(taxYear: Int,
 }
 
 object DeleteEmploymentAudit {
-  implicit def writes: OWrites[DeleteEmploymentAudit] = Json.writes[DeleteEmploymentAudit]
+  implicit def writes: Writes[DeleteEmploymentAudit] = (audit: DeleteEmploymentAudit) => {
+    Json.obj(
+      "taxYear" -> audit.taxYear,
+      "userType" -> audit.userType,
+      "nino" -> audit.nino,
+      "mtditid" -> audit.mtditid,
+      "employmentData" -> audit.employmentData
+    ).++(
+      audit.benefits.fold(Json.obj())(benefits => Json.obj("benefits" -> benefits))
+    ).++(
+      audit.expenses.fold(Json.obj())(expenses => Json.obj("expenses" -> expenses))
```

Comment:
Expenses should only contain 4 of the fields and not the entire object to match the audit event.
![Uploading Screenshot 2022-02-09 at 16.58.48.png…]()

Reply:
See toAuditModel function above on line 35

Reply:
PR has been updated. This should be resolved.